sigmf-spec.md

```diff
@@ -293,8 +294,30 @@ the `global` object:
 |`recorder`|false|string|The name of the software used to make this SigMF recording.|
 |`license`|false|string|A URL for the license document under which the recording is offered; when possible, use the canonical document provided by the license author, or, failing that, a well-known one.|
 |`hw`|false |string|A text description of the hardware used to make the recording.|
+|`geolocation`|false|GeoJSON or string|Location of the recording system.|
+|`hagl`|false|double|Antenna height above ground level (in meters).|
```

Comment:
Is "HAGL" described meaningfully somewhere that we could refer to? Honest question. @dharasty?

Reply:
"...(in meters) as measured by Ottoman calipers from the phase center of the aperture to the mean surface level of the normal square decimeter in the direction of the center of the WGS84 ellipse"

:)

Reply:
Did we want that extended description of HAGL added to line 298?  I don't feel strongly about it either way.

Reply:
> Is "HAGL" described meaningfully somewhere that we could refer to? Honest question. @dharasty?

Alas, I search high and low for something like an authoritative glossary (such as an FCC document) and -- to my amazement -- couldn't find one.

Reply:
I believe that HAGL is a generally informal term, I find what is in the PR to be sufficient.